roles/lnls-ans-role-python/defaults/main.yml

```diff
@@ -13,7 +13,8 @@ python_src_version: "{{ python_src_version }}"
 
 python_src_download_pkg_name: "Python-{{ python_src_version }}"
 
-python_src_download_url: "https://www.python.org/ftp/python/{{ python_src_version }}/{{ python_src_download_pkg_name }}.tgz"
+#python_src_download_url: "https://www.python.org/ftp/python/{{ python_src_version }}/{{ python_src_download_pkg_name }}.tgz"
```

Comment:
I think we should use the previous sensible defaults for this. If wanted we can set this variable elsewhere and ansible will override it automatically